ScoutSuite/providers/gcp/facade/iam.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+
+from googleapiclient import discovery
+from ScoutSuite.providers.gcp.utils import MemoryCache
+
+class IAMFacade:
+    def __init__(self):
+        self._iam_client = discovery.build('iam', 'v1', cache_discovery=False, cache=MemoryCache())
```

Comment:
[Same comment](https://github.com/nccgroup/ScoutSuite/pull/229#discussion_r263377080), I don't think this is the right place to do HTTP requests.